packages/yourloops/components/menus/team-menu.tsx

```diff
@@ -169,24 +191,30 @@ function TeamMenu(): JSX.Element {
           </ListItem>
         }
 
-        {isUserHcp &&
+        {isUserHcp || isUserPatient &&
```

Comment:
need brackets to properly check the condition
```suggestion
        {(isUserHcp || isUserPatient) &&
```